BrowserKit/Sources/Shared/UserAgent.swift

```diff
@@ -102,6 +102,7 @@ struct CustomUserAgentConstant {
     private static let safariMobileUA = UserAgentBuilder.defaultMobileUserAgent().clone(extensions: "Version/18.6 \(UserAgent.uaBitMobile) \(UserAgent.uaBitSafari)")
 
     static let customMobileUAForDomain = [
+        "tver.jp": safariMobileUA,
```

Comment:
Oh, actually, I do have a nit. Can we add a comment behind each line to indicate which site report bug this belongs to? So, esentially

```suggestion
        "tver.jp": safariMobileUA, // Bug 1983983
```

although I have not checked if this passes linting. The motivation here is to make it a lot easier to circle back to the bug if we need to, as opposed of going through the git blame > jira > bugzilla chain.

Reply:
Its more of a nit and we usually do vertical comments for our FXIOS bugs along with description

Reply:
To add on to @nbhasin2, we usually do the following, so would be nice to have an associated FXIOS ticket, we can also tag the bugzilla at the end
```
// TODO: FXIOS-XXX description on the issue [Bugzilla 1983983]
```

Reply:
I'm adding a note for this and the previous issue (below) too:

```suggestion
        // TODO: FXIOS-13391 [webcompat] "connection error" only on FxiOS/* UA (bug 1983983)
        "tver.jp": safariMobileUA,
        // TODO: FXIOS-13096 [webcompat] UA version parsed as "Safari 0" (webcompat #170304)
```

Feel free to patch up too, if you'd like a better comment for future reference. Thanks!